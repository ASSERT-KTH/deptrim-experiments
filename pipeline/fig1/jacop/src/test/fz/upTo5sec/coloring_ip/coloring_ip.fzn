array [1..3] of int: X_INTRODUCED_62 = [1,1,-1];
array [1..5] of int: X_INTRODUCED_164 = [1,1,1,1,1];
var 0..1: X_INTRODUCED_0;
var 0..1: X_INTRODUCED_1;
var 0..1: X_INTRODUCED_2;
var 0..1: X_INTRODUCED_3;
var 0..1: X_INTRODUCED_4;
var 0..1: X_INTRODUCED_5;
var 0..1: X_INTRODUCED_6;
var 0..1: X_INTRODUCED_7;
var 0..1: X_INTRODUCED_8;
var 0..1: X_INTRODUCED_9;
var 0..1: X_INTRODUCED_10;
var 0..1: X_INTRODUCED_11;
var 0..1: X_INTRODUCED_12;
var 0..1: X_INTRODUCED_13;
var 0..1: X_INTRODUCED_14;
var 0..1: X_INTRODUCED_15;
var 0..1: X_INTRODUCED_16;
var 0..1: X_INTRODUCED_17;
var 0..1: X_INTRODUCED_18;
var 0..1: X_INTRODUCED_19;
var 0..1: X_INTRODUCED_20;
var 0..1: X_INTRODUCED_21;
var 0..1: X_INTRODUCED_22;
var 0..1: X_INTRODUCED_23;
var 0..1: X_INTRODUCED_24;
var 0..1: X_INTRODUCED_25;
var 0..1: X_INTRODUCED_26;
var 0..1: X_INTRODUCED_27;
var 0..1: X_INTRODUCED_28;
var 0..1: X_INTRODUCED_29;
var 0..1: X_INTRODUCED_30;
var 0..1: X_INTRODUCED_31;
var 0..1: X_INTRODUCED_32;
var 0..1: X_INTRODUCED_33;
var 0..1: X_INTRODUCED_34;
var 0..1: X_INTRODUCED_35;
var 0..1: X_INTRODUCED_36;
var 0..1: X_INTRODUCED_37;
var 0..1: X_INTRODUCED_38;
var 0..1: X_INTRODUCED_39;
var 0..1: X_INTRODUCED_40;
var 0..1: X_INTRODUCED_41;
var 0..1: X_INTRODUCED_42;
var 0..1: X_INTRODUCED_43;
var 0..1: X_INTRODUCED_44;
var 0..1: X_INTRODUCED_45;
var 0..1: X_INTRODUCED_46;
var 0..1: X_INTRODUCED_47;
var 0..1: X_INTRODUCED_48;
var 0..1: X_INTRODUCED_49;
var 0..1: X_INTRODUCED_50;
var 0..1: X_INTRODUCED_51;
var 0..1: X_INTRODUCED_52;
var 0..1: X_INTRODUCED_53;
var 0..1: X_INTRODUCED_54;
var 0..1: X_INTRODUCED_55;
var 0..1: X_INTRODUCED_56;
var 0..1: X_INTRODUCED_57;
var 0..1: X_INTRODUCED_58;
var 0..1: X_INTRODUCED_59;
var 0..5: obj:: is_defined_var;
array [1..55] of var int: x:: output_array([1..11,1..5]) = [X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4,X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9,X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14,X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19,X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24,X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29,X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34,X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39,X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44,X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49,X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54];
array [1..5] of var int: u:: output_array([1..5]) = [X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59];
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_5,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_6,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_7,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_8,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_9,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_15,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_16,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_17,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_18,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_19,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_30,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_31,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_32,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_33,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_34,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_0,X_INTRODUCED_40,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_1,X_INTRODUCED_41,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_2,X_INTRODUCED_42,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_3,X_INTRODUCED_43,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_4,X_INTRODUCED_44,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_10,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_6,X_INTRODUCED_11,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_12,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_8,X_INTRODUCED_13,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_9,X_INTRODUCED_14,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_25,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_6,X_INTRODUCED_26,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_27,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_8,X_INTRODUCED_28,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_9,X_INTRODUCED_29,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_5,X_INTRODUCED_35,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_6,X_INTRODUCED_36,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_7,X_INTRODUCED_37,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_8,X_INTRODUCED_38,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_9,X_INTRODUCED_39,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_10,X_INTRODUCED_20,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_11,X_INTRODUCED_21,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_12,X_INTRODUCED_22,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_13,X_INTRODUCED_23,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_14,X_INTRODUCED_24,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_10,X_INTRODUCED_30,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_11,X_INTRODUCED_31,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_12,X_INTRODUCED_32,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_13,X_INTRODUCED_33,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_14,X_INTRODUCED_34,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_10,X_INTRODUCED_45,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_11,X_INTRODUCED_46,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_12,X_INTRODUCED_47,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_13,X_INTRODUCED_48,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_14,X_INTRODUCED_49,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_15,X_INTRODUCED_20,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_16,X_INTRODUCED_21,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_17,X_INTRODUCED_22,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_18,X_INTRODUCED_23,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_19,X_INTRODUCED_24,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_15,X_INTRODUCED_25,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_16,X_INTRODUCED_26,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_17,X_INTRODUCED_27,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_18,X_INTRODUCED_28,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_19,X_INTRODUCED_29,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_15,X_INTRODUCED_45,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_16,X_INTRODUCED_46,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_17,X_INTRODUCED_47,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_18,X_INTRODUCED_48,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_19,X_INTRODUCED_49,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_20,X_INTRODUCED_35,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_21,X_INTRODUCED_36,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_22,X_INTRODUCED_37,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_23,X_INTRODUCED_38,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_24,X_INTRODUCED_39,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_20,X_INTRODUCED_40,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_21,X_INTRODUCED_41,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_22,X_INTRODUCED_42,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_23,X_INTRODUCED_43,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_24,X_INTRODUCED_44,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_25,X_INTRODUCED_50,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_26,X_INTRODUCED_51,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_27,X_INTRODUCED_52,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_28,X_INTRODUCED_53,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_29,X_INTRODUCED_54,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_30,X_INTRODUCED_50,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_31,X_INTRODUCED_51,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_32,X_INTRODUCED_52,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_33,X_INTRODUCED_53,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_34,X_INTRODUCED_54,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_35,X_INTRODUCED_50,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_36,X_INTRODUCED_51,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_37,X_INTRODUCED_52,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_38,X_INTRODUCED_53,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_39,X_INTRODUCED_54,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_40,X_INTRODUCED_50,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_41,X_INTRODUCED_51,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_42,X_INTRODUCED_52,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_43,X_INTRODUCED_53,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_44,X_INTRODUCED_54,X_INTRODUCED_59],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_45,X_INTRODUCED_50,X_INTRODUCED_55],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_46,X_INTRODUCED_51,X_INTRODUCED_56],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_47,X_INTRODUCED_52,X_INTRODUCED_57],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_48,X_INTRODUCED_53,X_INTRODUCED_58],0);
constraint int_lin_le(X_INTRODUCED_62,[X_INTRODUCED_49,X_INTRODUCED_54,X_INTRODUCED_59],0);
constraint int_lin_eq(X_INTRODUCED_164,[X_INTRODUCED_0,X_INTRODUCED_1,X_INTRODUCED_2,X_INTRODUCED_3,X_INTRODUCED_4],1);
constraint int_lin_eq(X_INTRODUCED_164,[X_INTRODUCED_5,X_INTRODUCED_6,X_INTRODUCED_7,X_INTRODUCED_8,X_INTRODUCED_9],1);
constraint int_lin_eq(X_INTRODUCED_164,[X_INTRODUCED_10,X_INTRODUCED_11,X_INTRODUCED_12,X_INTRODUCED_13,X_INTRODUCED_14],1);
constraint int_lin_eq(X_INTRODUCED_164,[X_INTRODUCED_15,X_INTRODUCED_16,X_INTRODUCED_17,X_INTRODUCED_18,X_INTRODUCED_19],1);
constraint int_lin_eq(X_INTRODUCED_164,[X_INTRODUCED_20,X_INTRODUCED_21,X_INTRODUCED_22,X_INTRODUCED_23,X_INTRODUCED_24],1);
constraint int_lin_eq(X_INTRODUCED_164,[X_INTRODUCED_25,X_INTRODUCED_26,X_INTRODUCED_27,X_INTRODUCED_28,X_INTRODUCED_29],1);
constraint int_lin_eq(X_INTRODUCED_164,[X_INTRODUCED_30,X_INTRODUCED_31,X_INTRODUCED_32,X_INTRODUCED_33,X_INTRODUCED_34],1);
constraint int_lin_eq(X_INTRODUCED_164,[X_INTRODUCED_35,X_INTRODUCED_36,X_INTRODUCED_37,X_INTRODUCED_38,X_INTRODUCED_39],1);
constraint int_lin_eq(X_INTRODUCED_164,[X_INTRODUCED_40,X_INTRODUCED_41,X_INTRODUCED_42,X_INTRODUCED_43,X_INTRODUCED_44],1);
constraint int_lin_eq(X_INTRODUCED_164,[X_INTRODUCED_45,X_INTRODUCED_46,X_INTRODUCED_47,X_INTRODUCED_48,X_INTRODUCED_49],1);
constraint int_lin_eq(X_INTRODUCED_164,[X_INTRODUCED_50,X_INTRODUCED_51,X_INTRODUCED_52,X_INTRODUCED_53,X_INTRODUCED_54],1);
constraint int_lin_eq([1,1,1,1,1,-1],[X_INTRODUCED_55,X_INTRODUCED_56,X_INTRODUCED_57,X_INTRODUCED_58,X_INTRODUCED_59,obj],0):: defines_var(obj);
solve  minimize obj;
