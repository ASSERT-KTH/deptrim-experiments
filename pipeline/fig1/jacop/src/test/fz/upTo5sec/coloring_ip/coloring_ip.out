x = array2d(1..11, 1..5, [0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0]);
u = array1d(1..5, [0, 1, 1, 1, 1]);
----------
==========